src/styles/utils/_vars.scss

```diff
@@ -1 +0,0 @@
-$c-gray: #eee;
```

Comment:
It would be good to utilise variables especially for things like:
- colors
- font sizes
- line heights